pkg/virt-operator/webhooks/webhook.go

```diff
@@ -90,3 +90,82 @@ func (k *KubeVirtDeletionAdmitter) Admit(review *v1beta1.AdmissionReview) *v1bet
 
 	return validating_webhooks.NewPassingAdmissionResponse()
 }
+
+func NewKubeVirtCreationAdmitter(client kubecli.KubevirtClient) *KubeVirtCreationAdmitter {
```

Comment:
You need to register this like [here](https://github.com/kubevirt/kubevirt/blob/master/pkg/virt-operator/creation/components/webhooks.go#L47).